versions/versions.go

```diff
@@ -18,6 +18,7 @@ const (
 
 var (
 	releasePattern       = regexp.MustCompile(`^(\d+)\.(x|\d+\.\d+)$`)
+	patchPattern         = regexp.MustCompile(`^(\d+\.\d+\.\d+)-([\w\d]+)$`)
```

Comment:
I think we should start with a more restrictive regex and then maybe loosen it later. what about `...-patch\d+`
?

Reply:
Restricting it to just `-patch` would not satisfy my needs unfortunately 😞 Do you feel strongly that it should be restrictive?

Reply:
No, it's fine as-is.